docs/Configuration.md

```diff
@@ -193,7 +193,7 @@ Indicates which provider should be used to instrument code for coverage. Allowed
 
 Note that using `v8` is considered experimental. This uses V8's builtin code coverage rather than one based on Babel. It is not as well tested, and it has also improved in the last few releases of Node. Using the latest versions of node (v14 at the time of this writing) will yield better results.
 
-### `coverageReporters` [array\<string | [string,any]>]
+### `coverageReporters` [CoverageReporters]
```

Comment:
This type is not defined in the docs, do you think it'll be confusing for users?

Reply:
I did hesitate a bit editing this line because of the reason you've mentioned. I then searched docs and here in the file we also have a ProjectConfig type in a signature and as "projects" config field definition, although it is not "defined" itself. 

Several combinations of options shape make it tedious to list and not pleasant to maintain; and [string, any] type is not quite correct as we are about to become limited to these shapes. While I decided to use the custom TS type, it still is not terrible to leave it as it were before, imo. So what do you think?

Reply:
We could link to the types (code)?

Reply:
thanks for the tip, done.